src/renderers/native/ReactNativeFiberInspector.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactNativeFiberInspector
+ * @flow
+ */
+'use strict';
+
+const ReactNativeComponentTree = require('ReactNativeComponentTree');
+
+if (__DEV__) {
+  var traverseOwnerTreeUp = function(hierarchy, instance) {
+    if (instance) {
+      hierarchy.unshift(instance);
+      traverseOwnerTreeUp(hierarchy, instance._debugOwner);
+    }
+  };
+
+  var getOwnerHierarchy = function(instance) {
+    var hierarchy = [];
+    traverseOwnerTreeUp(hierarchy, instance);
+    return hierarchy;
+  };
+
+  var lastNotNativeInstance = function(hierarchy) {
+    for (let i = hierarchy.length - 1; i > 1; i--) {
+      const instance = hierarchy[i];
+      if (!instance.viewConfig) {
+        return instance;
+      }
+    }
+    return hierarchy[0];
+  };
+
+  var getInspectorDataForViewTag = function(viewTag: any): Object {
+    const fiber = ReactNativeComponentTree.findFiberByHostInstance(viewTag);
+    const hierarchy = getOwnerHierarchy(fiber);
+    const instance = lastNotNativeInstance(hierarchy);
+    const props = instance.memoizedProps || {};
```

Comment:
I'm not sure. :) The way we solved it in the devtools is that we track everything that could've been updated but it seems like overkill to do here.

We could use `ReactFiberTreeReflection.findCurrentFiberUsingSlowPath`. That one will guarantee to yield the correct props for this particular Fiber but won't yield the correct props for every Fiber on the way up to the root owner.

We could call it repeatedly for every Fiber on the way to the root. Maybe that's not too bad for this use case?